test/functional/test_site/testPlantUML.md

```diff
@@ -0,0 +1,22 @@
+## PlantUML Test
+
+### Sequence Diagram
+<puml src="diagrams/sequence.puml" />
```

Comment:
When I generate this page, the `src` will be changed to `test_site/diagrams/abc.png` when the correct one should be without `test_site`.

Reply:
Seems to be an issue with the `<pic>` component, will look into it